x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -230,7 +230,7 @@ export const policyFactoryWithoutPaidFeatures = (
         ...policy.mac.popup,
         malware: {
           message: '',
-          enabled: true,
+          enabled: policy.mac.popup.malware.enabled,
```

Comment:
@gergoabraham we should still not change this when we are looking at an unpaid license because part of the advantage of the license is that users can customize the message and toggle it on/off.

This is what the UI looks like with a basic feature:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/56395104/198685836-22236a46-8b94-4fa5-a890-c59e4c3980c0.png">

Reply:
@kevinlog thanks for the explanation, it makes sense now!

I've reverted the modification with an explanatory comment: 5b30c69

And switched the order of the logic, so license limitations are applied in the final step, so `disableProtections` won't override and disable malware popup: 570c16d

(Disabling malware popup when disabling protections shouldn't be a problem in practice though, I think, because after enabling the malware protection, the popup will be automatically enabled, but still, I think it is clearer to apply the limitations at the end.)

Reply:
Thanks @gergoabraham !  The logic makes sense to me, we should just be sure to test the cases using the existing NGAV, EDR Essential, and EDR Complete with a basic license and ensure that everything is still working.